### PR TITLE
Item cards

### PR DESCRIPTION
**Pull request by rishnarenikunta**
Inventory Card and Shopping Card

### CODE REVIEW

**Comment by emoryjgrubbs** [comment]
Remove the prisma migration files with 'git rm -r --cached prisma' and commit that. That should just removed it from the repo and not your local files.

I would make the type all caps just to keep it consistent, but doesn't matter too much. Maybe we should figure out standard type names and document that somewhere.

make the div on 29 an input, even if it doesn't connect to anything for now, it's good to make sure it's still styled correctly/you're getting the input.

I don't think the tailwind h/w is doing anything right now. it's not defined in the tailwind docs, and, for me at least, the image is not being scaled up. (w-64 does 256 pixels, and same for height)

You might expand the count/change bar to line up with the outside of the image, looks a little small right now for me.

You might put a drop shadow on the three dots menu to help it differentiate on light background images.

Probably at least add an issue for closing the three dots menu when clicking outside it.

I would probably also only put the things changed by whether it's shopping or inventory in the v-if, so that it's easier to maintain.

My output with changed width:
image
 
My output without changed width:
image

**Comment by rishnarenikunta** [comment]
Fixed the changes requested for the item card component. The new cards look like this: 
Screenshot 2024-10-22 at 11 30 57 PM

File: prisma/migrations/20240829204639_init/migration.sql
Comment (CrazEpic) [inline prisma/migrations/20240829204639_init/migration.sql:1]: delete this

File: prisma/migrations/migration_lock.toml
Comment (CrazEpic) [inline prisma/migrations/migration_lock.toml:1]: delete this file

**Review by CrazEpic — APPROVED** [review]
looks good, just a few nits about coding style, also will need to update the API calls once we merge to stage

additionally, we should try displaying the items on their respective pages so that a reviewer doesn't have to change the code to see shopping vs inventory card views

**Comment by emoryjgrubbs** [comment]
similar comments to daniel, the migration files got re-added so you should remove them and add them to a .gitignore. Also, this is on me, but you'll want to make sure to just do it for the migrations and not the seed/schema files.

add to cart text looks a little thin to me, maybe bump it to semibold

in the future, i would just do one div (can have whatever amount of styling would apply to all of it, including none, that does the v-if, but it's good for now

i'd also suggest putting the parenthesizes with any functionality before the styling as it's more important to read.

**Pull request merged**